### PR TITLE
Enable provisioning wizard in Ephemeral environment

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

**Comment by lucasgarfield** [comment]
Hi Ondrej, I'm still trying to get my head around the useScalprum() and useChrome() hooks. Just for my own knowledge, would you mind explaining a bit about why these changes are needed for the ephemeral environment? How did you know that you needed to make these changes?

File: src/Components/ImagesTable/ImageLink.js [hunk 1]
```diff
@@ -62,12 +63,13 @@ const ProvisioningLink = ({ imageId, isExpired, isInClonesTable }) => {
 const ImageLink = ({ imageId, isExpired, isInClonesTable }) => {
   const image = useSelector((state) => selectImageById(state, imageId));
   const uploadStatus = image.uploadStatus;
+  const { initiated: chromeInitiated, isBeta, getEnvironment } = useChrome();
```
Comment (lucasgarfield) [inline src/Components/ImagesTable/ImageLink.js:66]: I think `initiated` is a typo, shouldn't it be `initialized`? 

Doesn't work for me with `initiated` but does work with `initialized`...

**Review by lucasgarfield — CHANGES_REQUESTED** [review]

**Comment by akhil-jha** [comment]
@ezr-ondrej can you update it please? :) I am not able to open the wizard through "Launch" yet. :)

**Comment by lucasgarfield** [comment]
Superseded by #926.

**Pull request closed without merging**